src/libs/antares/study/fwd.h

```diff
@@ -214,16 +214,18 @@ enum TimeSeries : unsigned int
     //! TimeSeries : Wind
     timeSeriesWind = 4,
     //! TimeSeries : Thermal
-    timeSeriesThermal = 8,
+    timeSeriesThermal = 128,
```

Comment:
These enums are used for switch cases and for bitwise operations and I don't thinks so that it will break something, but I will try with constant defined outside, to see if that will break something.

Reply:
If we define timeSeriesCount outside from enum we will break compilation of solver, but that compilation errors could be resolved easily. But also, defining timeSeriesCount outside of enum will break compilation of the old UI, which couldn't be easily resolved, I think so.

Reply:
@nikolaredstork : I understand why you did that, but we cannot keep it that way.
So I fixed it.